### PR TITLE
Rustfmt and sort all `use` items

### PR DESCRIPTION
**Pull request by mcarton**
This is only `git add -p` bits, rustfmt cannot be trusted yet, it removed some comments and made some ugly stuff :disappointed:.

### CODE REVIEW

File: src/consts.rs [hunk 1]
```diff
@@ -224,7 +226,7 @@ fn constant_not(o: Constant) -> Option<Constant> {
                 }  // refuse to guess
             };
             Some(Int(!value & mask, LitIntType::Unsigned(ity), Sign::Plus))
-        },
+        }
```
Comment (oli-obk) [inline src/consts.rs]: is this the style we want? if no, we need to adjust the rustfmt config

Comment (mcarton) [inline src/consts.rs]: I’m ok with no comma here

File: src/loops.rs [hunk 2]
```diff
@@ -438,8 +438,12 @@ fn check_for_loop_reverse_range(cx: &LateContext, arg: &Expr, expr: &Expr) {
...
                 // smaller value.
                 let (sup, eq) = match (start_idx, stop_idx) {
-                    (ConstVal::Int(start_idx), ConstVal::Int(stop_idx)) => (start_idx > stop_idx, start_idx == stop_idx),
-                    (ConstVal::Uint(start_idx), ConstVal::Uint(stop_idx)) => (start_idx > stop_idx, start_idx == stop_idx),
+                    (ConstVal::Int(start_idx), ConstVal::Int(stop_idx)) => {
+                        (start_idx > stop_idx, start_idx == stop_idx)
+                    }
+                    (ConstVal::Uint(start_idx), ConstVal::Uint(stop_idx)) => {
+                        (start_idx > stop_idx, start_idx == stop_idx)
+                    }
```
Comment (oli-obk) [inline src/loops.rs]: is the max-line-length configured to 100? IIRC clippy has 120 or sth

Comment (mcarton) [inline src/loops.rs]: The `rustfmt.toml` file has
```toml
max_width = 120
ideal_width = 100
```

File: src/consts.rs (hunk 1 above)
Comment (Manishearth) [inline src/consts.rs]: same

**Pull request merged**